### DISCUSSION TITLE
Zigbee-herdsman debug logging

### DISCUSSION THREAD

**Discussion by LemmeDasker** [Koenkk/zigbee2mqtt#18829] [General]
I am trying to enable Zigbee-herdsman debug logging to help figure out why my device comes with an error.

I am on Windows, and trying to start Zigbee2MQTT with: C:\Zigbee2MQTT\zigbee2mqtt>DEBUG=zigbee-herdsman* npm start

But it fails with:
`'DEBUG' is not recognized as an internal or external command, operable program or batch file.`

What could be wrong?

**Comment by Nardol** [comment]
This syntax is OK for Linux but not for Windows it seams.
Maybe it would work as following:

```
C:\Zigbee2MQTT\zigbee2mqtt>set DEBUG=zigbee-herdsman*
C:\Zigbee2MQTT\zigbee2mqtt>npm start
```

After stopping Zigbee2MQTT type the following when you want to disable debug logging:
`C:\Zigbee2MQTT\zigbee2mqtt>set DEBUG=`
To be sure things will run as usual.

**Comment by LemmeDasker** [comment]
Perfect - many thanks! That worked :-)
